IMPLEMENTATION.adoc

```diff
@@ -2,65 +2,70 @@
 
 This document describes the current state of the implementation of the Kiali Operator (henceforce, "KO"). The purpose for this document is to provide an introduction to the internals of the KO to those who have a need to modify or enhance the KO.
 
-NOTE: This document reflects the current implementation as of release link:https://github.com/kiali/kiali-operator/tree/v1.69.0[v1.69.0]
+NOTE: This document reflects the current implementation as of release link:https://github.com/kiali/kiali-operator/tree/v1.76.0[v1.76.0]
```

Comment:
This branch doesn't exist yet.  In general, some entries here are for 1.76, some for 1.75 and some for master, is that all intentional?

Reply:
right, I just put that in there for now- we'll get that branch in a week